python/ray/_private/utils.py

```diff
@@ -1910,3 +1920,15 @@ def run_background_task(coroutine: Coroutine) -> asyncio.Task:
     # completion:
     task.add_done_callback(background_tasks.discard)
     return task
+
+
+def try_import_each_module(module_names_to_import: List[str]) -> None:
+    """
+    Make a best-effort attempt to import each named Python module.
+    This is used by the Python default_worker.py to preload modules.
+    """
+    for module_to_preload in module_names_to_import:
+        try:
+            importlib.import_module(module_to_preload)
+        except ImportError as e:
+            print(f'Failed to import "{module_to_preload}"', e)
```

Comment:
Can you use logger instead?

Reply:
Sure. Is there any danger using print vs logger at this stage? I saw the comment about how separate functions in this file impact some tensorflow workload and got a bit nervous to try something new.

Reply:
I meant in default worker 

(On mobile right now, sry)

if __name__ == "__main__":
    # NOTE(sang): For some reason, if we move the code below
    # to a separate function, tensorflow will capture that method
    # as a step function. For more details, check out
    # https://github.com/ray-project/ray/pull/12225#issue-525059663.

Reply:
I believe this one should be fine. Also we should always use the logger for internal code.